sources/packages/api/src-sql/AuthCodeTables/Create-institution-user-type.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE IF NOT EXISTS institution_user_type_roles(
+  id SERIAL PRIMARY KEY,
+  user_type_description VARCHAR(500) NULL,
+  user_role_description VARCHAR(500) NULL,
```

Comment:
Can we please increase the size of the user_role_description to more like 1000, I guess postgres supports more than 8000 limit too.

Reply:
yes, we can do that the actual limit is around 2^n - 1, where n is size of unicode char buffer.

Reply:
lets discuss and fix some good number for these kind of fields